src-docs/src/views/datagrid/datagrid_example.js

```diff
@@ -427,7 +427,7 @@ export const DataGridExample = {
               explanation on the lower level object types. The majority of the
               types are defined in the{' '}
               <a
-                href="https://github.com/elastic/eui/tree/master/src/components/datagrid/data_grid_types.ts"
```

Comment:
Could this elastic link also be changed to an OpenSearch link?

Reply:
Done